rules/do-you-have-a-good-page-speed/rule.md

```diff
@@ -1,18 +1,40 @@
 ---
 type: rule
-archivedreason: 
 title: Do you have a good page speed?
-guid: fdd7cf8b-d165-4010-9e96-481d3bd406b9
-uri: do-you-have-a-good-page-speed
-created: 2019-04-16T22:59:21.0000000Z
+uri: website-page-speed
```

Comment:
Hi @camillars 

Because you changed the URI, you need to action 3 more steps:
1. Update the folder name to be the same as the new URI
2. Add a redirect for the old one
3. Update the URI in the category(s) the rule is under

More info on the Rules.Content wiki